tools/cldr-code/src/main/java/org/unicode/cldr/util/CodePointEscaper.java

```diff
@@ -10,80 +10,99 @@
 
     // Spaces
-    SP(0x20, "space", "ASCII space"),
-    TSP(0x2009, "thin space", "Aka ‘narrow space’"),
+    TSP(0x2009, "Thin space", "Aka ‘narrow space’"),
 
     // No-break versions
-    NBSP(0xA0, "no-break space", "Same as space, but doesn’t line wrap."),
+    NBSP(0xA0, "No-break space", "Same as space, but doesn’t line break."),
     NBTSP(
             0x202F,
-            "no-break thin space",
-            "Same as thin space, but doesn’t line wrap. Aka 'narrow no-break space'"),
+            "No-break thin space",
+            "Same as thin space, but doesn’t line break. Aka 'narrow no-break space'"),
 
     // Line Break control
-    WNJ(
+    ALB(
             0x200B,
-            "allow line wrap after, aka ZWSP",
-            "Invisible character allowing a line-wrap afterwards. Also known as ‘ZWSP’."),
-    WJ(
-            0x2060,
-            "prevent line wrap",
-            "Keeps adjacent characters from line-wrapping. Also known as ‘word-joiner’."),
+            "Allow line break",
+            "Invisible character allowing a line-break after. Also known as ‘ZWSP’ (zero-width space)."),
+    NB(0x2060, "No-break", "Prevents line break before or after. Also known as ‘word-joiner’."),
+
+    // Hyphens & dashes
+    NBHY(
+            0x2011,
+            "No-break hyphen",
+            "Same as a hyphen, but disallows line break after as well as before."),
+
     SHY(
             0x00AD,
-            "soft hyphen",
+            "Soft hyphen",
             "Invisible character allowing a line-wrap afterwards, but appears like a hyphen in most languages."),
 
-    ZWNJ(0x200C, "cursive non-joiner", "Breaks cursive connections, where possible."),
-    ZWJ(0x200D, "cursive joiner", "Forces cursive connections, if possible."),
+    NDASH(
+            0x2013,
+            "En dash",
+            "Slightly wider (–) than a hyphen (-), used for ranges of numbers or dates in some languages."),
+
+    // Cursive joining
+
+    ZWNJ(0x200C, "Cursive non-joiner", "Breaks cursive connections, where possible."),
+    ZWJ(0x200D, "Cursive joiner", "Forces cursive connections, if possible."),
+
+    // Bidi
+
+    LRM(
+            0x200E,
+            "Left-right mark",
+            "For BIDI, invisible character that behaves like Hebrew letter."),
+    RLM(0x200F, "Right-left mark", "For BIDI, invisible character that behaves like Latin letter."),
 
     ALM(
             0x061C,
             "Arabic letter mark",
             "For BIDI, invisible character that behaves like Arabic letter."),
-    LRM(
-            0x200E,
-            "left-right mark",
-            "For BIDI, invisible character that behaves like Hebrew letter."),
-    RLM(0x200F, "right-left mark", "For BIDI, invisible character that behaves like Latin letter."),
 
-    LRO(0x202D, "left-right override"),
-    RLO(0x202E, "right-left override"),
-    PDF(0x202C, "end override"),
+    // Oddball Cf characters that are in CLDR data
```

Comment:
```suggestion
    // Other Cf characters that are in CLDR data
```

Are they here because they are Cf?  The only practical reason I could think of to include them is confusabillity (that ؂ might look like another character, etc)

Reply:
Yes